server/plugin.go

```diff
@@ -227,7 +228,7 @@ func (p *Plugin) storeGitHubUserInfo(info *GitHubUserInfo) error {
 		return err
 	}
 
-	if err := p.API.KVSet(info.UserID+GITHUB_TOKEN_KEY, jsonInfo); err != nil {
+	if err := p.API.KVSet(info.UserID+githubTokenKey, jsonInfo); err != nil {
 		return err
```

Comment:
Shouldn't we wrap this error?

Reply:
In general I fully support error wrapping. But I'm inclined to deffer the topic to another PR given how many lines this PR touches. 0/5

Reply:
That's a good point. I agree

Reply:
Opened: https://github.com/mattermost/mattermost-plugin-github/issues/254